caffe2/core/blob.h

```diff
@@ -286,7 +280,7 @@ class CAFFE2_API Blob {
   }
   TypeMeta meta_;
   void* pointer_ = nullptr;
-  DestroyCall destroy_ = nullptr;
+  DestroyCall* destroy_ = nullptr;
```

Comment:
`const DestroyCall*`?

Reply:
tbh, I'm not sure what a const function pointer would be semantically. It's not like a function object is mutable and you can change its implementation at runtime.